### PR TITLE
add working-with-prometheus-in-control-plane

### PR DESCRIPTION
**Pull request by Charlie17Li**
Signed-off-by: charlie <qianglin98@qq.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
add the doc for "Use Prometheus to monitor Karmada control plane"

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by Charlie17Li** [comment]
cc @Poor12

File: docs/administrator/monitoring/working-with-prometheus-in-control-plane.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+       kubernetes.io/service-account.name: "prometheus"
+   ```
+
+3. Get the token for accessing the karmada apiserver
+
+   ```shell
+   kubectl get secret prometheus -o=jsonpath={.data.token} --context "karmada-apiserver" | base64 -d
+   ```
+
+4. Create resource objects of Prometheus, also you need replace `<karmada-token>` (2 places) with the token got from step 3
```
Comment (Poor12) [inline docs/administrator/monitoring/working-with-prometheus-in-control-plane.md:100]: Can you mark these two places clearly？

File: docs/administrator/monitoring/working-with-prometheus-in-control-plane.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+             name: prometheus-config
+         - name: prom-data
+           hostPath:
+             path: /var/lib/prom-data
+             type: DirectoryOrCreate
+   
+   ```
+
+3. Use any node IP of the control plane and the port number (default 31801) to enter the Prometheus monitoring page of the control plane
+
```
Comment (Poor12) [inline docs/administrator/monitoring/working-with-prometheus-in-control-plane.md:269]: BTW，Can you describe how to access Grafana and show the Grafana control panel？

Comment (Charlie17Li) [reply docs/administrator/monitoring/working-with-prometheus-in-control-plane.md:269]: done

File: docs/administrator/monitoring/working-with-prometheus-in-control-plane.md (hunk 1 above)
Comment (Charlie17Li) [reply docs/administrator/monitoring/working-with-prometheus-in-control-plane.md]: OK

**Review by Poor12 — COMMENTED** [review]
Generally looks good to me.
cc @RainbowMango

File: sidebars.js [hunk 3]
```diff
@@ -152,6 +152,7 @@ module.exports = {
                     items: [
                         "administrator/monitoring/working-with-filebeat",
                         "administrator/monitoring/working-with-prometheus",
+                        "administrator/monitoring/working-with-prometheus-in-control-plane"
```
[the remark this answers is not in the document]
Comment (Charlie17Li) [reply sidebars.js]: OK

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**